sdk/core/core-http/src/policies/exponentialRetryPolicy.ts

```diff
@@ -245,6 +187,6 @@ function retry(
       );
     return Promise.reject(err);
```

Comment:
for consistency should this be `throw err` ?

Reply:
Fixed